doc_source/get-set-up-for-aws-batch.md

```diff
@@ -95,7 +95,7 @@ For more information about IAM, see the [AWS Identity and Access Management User
 
 ## Create IAM Roles for your Compute Environments and Container Instances<a name="create-an-iam-role"></a>
 
-Your AWS Batch compute environments and container instances require AWS account credentials to make calls to other AWS APIs on your behalf\. You must create IAM roles that provides these credentials to your compute environments and container instances and then associate that role with your compute environments\.
+Your AWS Batch compute environments and container instances require AWS account credentials to make calls to other AWS APIs on your behalf\. You must create IAM roles that provide these credentials to your compute environments and container instances, then associate those roles with your compute environments\.
```

Comment:
Sorry, I missed the nuance here on the first look. Apologies. Please make this "create an IAM role" "associate that role".